src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -605,6 +603,34 @@
                     ]
                 }
             ]
+        },
+        {
+            "path": "/v1/debug/node_uuid",
```

Comment:
Nit: a bit of name name inconsistency with node <-> broker. Also, maybe we can underscore that this is referring to _this_ node in the method name. It is okay if the current naming is consistent with other debug endpoints though.

Reply:
it is indeed consistent, do you think that `current_broker_uuid` would be better ?